rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -175,6 +233,13 @@ private static UTXO utxoOf(BtcTransaction bitcoinTransaction, TransactionOutput
         );
     }
 
+    private void setupChainWithBtcTransaction(BtcTransaction bitcoinTransaction) throws BlockStoreException {
+        pmtWithTransactions = createValidPmtForTransactions(List.of(bitcoinTransaction.getHash()), btcNetworkParams);
+        btcBlockWithPmtHeight = bridgeConstants.getBtcHeightWhenPegoutTxIndexActivates() + bridgeConstants.getPegoutTxIndexGracePeriodInBtcBlocks();
+        int chainHeight = btcBlockWithPmtHeight + bridgeConstants.getBtc2RskMinimumAcceptableConfirmations();
+        recreateChainFromPmt(btcBlockStoreWithCache, chainHeight, pmtWithTransactions, btcBlockWithPmtHeight, btcNetworkParams);
+    }
```

Comment:
Does it make sense to have this be changing state class per test? Would it be possible or does it make sense to return assigments to local variables per test?

Reply:
You mean having pmtWithTransactions and btcBlockWithPmtHeight as local variables in each test?

Reply:
Yes, it seems that it makes more sense to have this at per test level rather than at a per class level.